tests/test_tensorflowjs.py

```diff
@@ -6,13 +6,13 @@ class TestTensorflowjs(unittest.TestCase):
     def test_version(self):
         result = subprocess.run([
                 'tensorflowjs_converter',
-                '--version',
```

Comment:
See https://b.corp.google.com/issues/301473408#comment3 for why I made this change, version is trying to load __version__ from some Keras lib, but that doesn't exist so it exits with code 1.